### PR TITLE
add link to select c# language version

### LINKED ISSUE #7123: How can I use C# version 7.3?
I'm running this example with dotnet version 2.1.400 in VSCode and get the warning  "CS8371: Field-targeted attributes on auto-properties are not supported in language version 7.0. Please use language version 7.3 or greater."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b7bc5cfb-d247-62c8-2173-9403efe72b16
* Version Independent ID: 3fab702d-66a7-3338-6c61-f535b29d86a7
* Content: [Walkthrough: Persisting an Object using C#](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio)
* Content Source: [docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by Niladri24dutta**
## Summary

Add reference link to select required C# language version for the sample code.

Fixes #7123

### CODE REVIEW

File: docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ In this walkthrough, you will create a basic `Loan` object and persist its data
 > [!TIP]
 > Need to install a code editor? Try [Visual Studio](https://visualstudio.com/downloads)!
 
+* The below code example requires C# 7.3 to be selected as the language version. Please refer to below
```
Comment (rpetrusha) [inline docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md]: - The below example code requires C# 7.3 to be selected as the language version. --> The example requires C# 7.3.

- Please refer... --> See \[Select the C# language version](../../../language-reference/configure-language-version.md).

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for contributing to the dotnet/docs repo, @Niladri24dutta, and adding information about selecting the language version. I've made some suggestions to condense the text.

File: docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md (hunk 1 above)
Comment (Niladri24dutta) [reply docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md]: OK I will do the required changes

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @Niladri24dutta, for making the additional changes. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Comment by Niladri24dutta** [comment]
@rpetrusha Thanks a lot.

**Pull request merged**